crates/katana/runner/src/deployer.rs

```diff
@@ -0,0 +1,50 @@
+use anyhow::{Context, Ok, Result};
+use starknet::core::types::FieldElement;
+use tokio::process::Command;
+
+use crate::KatanaRunner;
+
+impl KatanaRunner {
+    /// Known issue - rpc set in Scarb.toml overrides command line argument
+    pub async fn deploy(&self, manifest: &str, script: &str) -> Result<FieldElement> {
+        let rpc_url = &format!("http://localhost:{}", self.port);
+
+        let out = Command::new("sozo")
+            .arg("migrate")
+            .args(["--rpc-url", rpc_url])
+            .args(["--manifest-path", manifest])
+            .output()
+            .await
+            .context("failed to start subprocess")?;
+
+        if !out.status.success() {
+            return Err(anyhow::anyhow!("deploy failed {:?}", out));
+        }
+
+        let constract_address = String::from_utf8_lossy(&out.stdout)
```

Comment:
ideally we can use json logs and parse those instead. it will be much more robust that this approach. i worry that this can easily break if sozo outputs change. maybe we should use sozo ops directly from benches

Reply:
Run as sozo ops, and extracted from the contract address from the output